README.md

```diff
@@ -72,6 +72,23 @@ For eclipse, you can generate eclipse project files by running `./gradlew eclips
 
 The latest pravega releases can be found on the [Github Release](https://github.com/pravega/pravega/releases) project page.
 
+## Snapshot artifacts 
+
+All snapshot artifacts from `master` and `release` branches are available in GitHUB Packages Registry 
+
+Add the following to your repositories list and import dependencies as usual. 
+
+```
+maven {
+    url "https://maven.pkg.github.com/pravega/pravega"
+    credentials {
+        username = "pravega-public"
+        password = "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"
```

Comment:
We need to encode this string in a way that GitHub won't automatically decode. GitHub protects against the public exposition of personal access tokens, so we need to obfuscate the string to prevent GitHub's detection methods.

Reply:
Yeah, encoded the new token